.github/workflows/ci_docker.yml

```diff
@@ -99,13 +99,37 @@ jobs:
           fi
 
       # If only this workflow file is changed, there is no need to publish Docker images.
-      - name: Push Docker images to Docker Hub
+      - name: Push x86 Docker images to Docker Hub
         if: |
           success()
           && steps.check-workflow-diff.outputs.changed_only == 'no'
           && steps.match-branch.outputs.match == 'true'
         working-directory: docker
-        run: bash +x publish.sh
+        run: bash +x publish-by-arch.sh --tag --publish
+        env:
+          DOCKER_USER: ${{ secrets.DOCKER_USER }}
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+
+      # If only this workflow file is changed, there is no need to publish Docker images.
+      - name: Push aarch64 Docker images to Docker Hub
+        if: |
+          success()
+          && steps.check-workflow-diff.outputs.changed_only == 'no'
+          && steps.match-branch.outputs.match == 'true'
+        working-directory: docker
+        run: bash +x publish-by-arch.sh --tag --buildx --publish
+        env:
+          DOCKER_USER: ${{ secrets.DOCKER_USER }}
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
It is recommended to use a matrix for this job [^1].
[^1]: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs